problems/finger_pointing_and_blaming.md

```diff
@@ -0,0 +1,48 @@
+---
+slug: finger-pointing-and-blaming
+stages:
+  - research
+short_description: Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust.
+tags:
+  - finger-pointing
+  - blaming
+  - culture
+  - mentality
+  - trust
+keywords:
+  - team player
+  - problem
+  - satisfaction
+  - responsibilities
+  - leadership
+  - help
+  - performance
+  - environment
+  - efficiency
+  - empowerment
+---
+
+# Finger-Pointing And Blaming
+
+**TL;DR**
+
+Finger-Pointing and Blaming is the act of blaming someone for a problem instead of trying to fix or take accountability. This results in a toxic work culture where nobody feels safe and there's no trust. It is important to combat this problem quickly in order to establish a healthy and a more efficient work environment. Read more to find more about this!
+
+## Impact Of The Blame Culture
+
+Watching someone pawn their failures off or blame others for their mistakes is contagious and detrimental, particularly in the workplace. It may seem inconsequential but in an organization where blame is the norm, people are likely to be less creative, perform poorly, and mirror damaging behaviors to protect their own self-image.
+When someone is always pointing the finger for their mistakes they can’t learn from those mistakes and hinders their own ability to learn to become more effective. A culture of blame quickly creates dissatisfied employees whose productivity will begin to slip.
+
+## Tips To Eliminate Blame Culture
+
+- At the first sign that there’s a problem, set up ground rules for positive, productive TEAM discussions. Explain that you’re not looking to assign blame and that any finger-pointing is forbidden.
```

Comment:
I would also suggest rewriting this part. It seems that there is a lot of similarities with the following: 
_At the first sign that there’s a problem, set up ground rules for positive, productive TEAM discussions. Explain that you’re not looking to assign blame and that any finger-pointing is forbidden.
Focus on PRODUCTIVE dialogue, and eliminate accusatory or negative language.
Get the team to focus on determining WHAT—not WHO—caused the problem.
Take an approach that centers around FIXING the problem instead of assigning blame for its occurrence.
Talk about what can be LEARNED from the problem.
Make sure you discuss how the team can AVOID a repeat of the problem._ from https://hrdailyadvisor.blr.com/2015/11/16/6-important-ways-to-avoid-playing-the-blame-game/